src/redux/greeting.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+const RANDOM_GREETING = 'greeting/RANDOM_GREETING';
+const baseurl = 'http://127.0.0.1:3000/api/greetings';
```

Comment:
Since data is crossing from one domain to another, you should consider adding cors to define where data should be accepted from.

![image](https://user-images.githubusercontent.com/27709832/228236073-e65b790e-d591-4720-a957-125b1909bda5.png)